examples/with-mobx/store.js

```diff
@@ -0,0 +1,30 @@
+import { action, observable } from 'mobx'
+
+class Store {
+  @observable lastUpdate = 0
+  @observable light = false
+
+  constructor (isServer, lastUpdate) {
+    this.lastUpdate = lastUpdate
+  }
+
+  @action start = () => {
+    this.timer = setInterval(() => {
+      this.lastUpdate = Date.now()
+      this.light = true
+    })
+  }
+
+  stop = () => clearInterval(this.timer)
+}
+
+export const initStore = (isServer, lastUpdate = Date.now()) => {
+  if (isServer && typeof window === 'undefined') {
+    return new Store(isServer, lastUpdate)
+  } else {
+    if (!window.store) {
```

Comment:
Do not assign the store into window.? You can save it as a local variable inside this module.
(If this because to support Mobx devtools, then that fine.)